### ISSUE TITLE
Question/feature request: Show open tickets on main public page

### ISSUE THREAD

**Issue by michaelzap** [helpdesk-z/helpdeskz-dev#44]
Is there a way to list all open tickets on the home page (with the KB articles)?

**Comment by Github-Minghui** [comment]
That might cause a serious security issue if customers provide confidential info in the tickets.

To showcase some of the frequently asked questions/tickets, you can document them and make a KB article for future customers.

**Comment by michaelzap** [comment]
Yes, I can see how that would be true if you're using this system for customer support.

I am actually trying to make it work for a different use case, which is why I was thinking that this might be an option (but not the default behavior). I would like to have this software on a private (in this case, password-protected) server and use it mostly for bug reports and feature requests for our organization's website and other tech. The idea is that say the development department might open a ticket asking if we could have peer-to-peer fundraising on our website and then that ticket would be visible as a feature request that I could reply to and everyone at our organization could view its status.

This may be something that I would need to manually code into the index page, and if so perhaps you could give me some tips for doing that. You might also consider whether this sort of use case (basically feature requests as opposed to help tickets) might be common enough to merit working it into the system. Perhaps each ticket would need to be approved before it appears?

Thanks for the great program!
